images/macos/toolsets/toolset-15.json

```diff
@@ -4,40 +4,82 @@
         "x64": {
             "versions": [
                 {
-                "link": "16.3_beta_2", "version": "16.3.0-Beta.2+16E5121h", "symlinks": ["16.3"], 
-                    "install_runtimes":
-                    [
-                        "iOS -buildVersion 18.0", "iOS -buildVersion 18.1", "iOS -buildVersion 18.2", "iOS -buildVersion 18.3.1", "iOS -buildVersion 22E5216h",
-                        "watchOS -buildVersion 11.0", "watchOS -buildVersion 11.1", "watchOS -buildVersion 11.2", "watchOS -buildVersion 22T5212l",
-                        "tvOS -buildVersion 18.0", "tvOS -buildVersion 18.1", "tvOS -buildVersion 18.2", "tvOS -buildVersion 22L5218l"
-                    ],
-                    "sha256": "f4bb80c2e93d3561ecbabc82d1a92c830d831c96afc48944baa8116e3fd0013d"
+                    "link": "16.3_beta_2", 
+                    "version": "16.3.0-Beta.2+16E5121h", 
+                    "symlinks": ["16.3"],
+                    "sha256": "f4bb80c2e93d3561ecbabc82d1a92c830d831c96afc48944baa8116e3fd0013d",
+                    "install_runtimes": [
+                        { "iOS": ["18.0", "18.1", "18.2", "18.3.1", "22E5216h"] },
+                        { "watchOS": ["11.0", "11.1", "11.2", "22T5212l"] },
+                        { "tvOS": ["18.0", "18.1", "18.2", "22L5218l"] }
+                    ]
+                },
+                {
+                    "link": "16.2",
+                    "version": "16.2+16C5032a",
+                    "sha256": "0e367d06eb7c334ea143bada5e4422f56688aabff571bedf0d2ad9434b7290de",
+                    "install_runtimes": ["none"]
+                },
+                {
+                    "link": "16.1",
+                    "version": "16.1+16B40",
+                    "sha256": "8ca961d55981f983d21b99a95a6b0ac04905b837f6e11346ee86d28f12afe720",
+                    "install_runtimes": ["none"]
```

Comment:
There seems to be multiple way on how to specify the things:
- `"install_runtimes": ["none"]`
- `"install_runtimes": "all"`
- `{ "iOS": ["18.0", "18.1", "18.2", "18.3.1", "22E5216h"] }`
- `{ "iOS": "18.2" }`

Should we standardize on when string constants should be used in place of objects and arrays?

I think it's ok to support both `{ "iOS": "18.2" }` and `{ "iOS": ["18.2"] }`

But the notation `"install_runtimes": ["none"]` feels a bit strange. I mean the `array of "none"`

Reply:
> I think it's ok to support both { "iOS": "18.2" } and { "iOS": ["18.2"] }
> But the notation "install_runtimes": ["none"] feels a bit strange. I mean the array of "none"

It will also work in both ways. Array of one string and one string are just the same for PS script, yeah. 
The only reason why I left arrays in json, even of one element, is so that everyone understands that there can be an arbitrary number of elements.